src/main/java/seedu/geekeep/logic/commands/FindCommand.java

```diff
@@ -11,20 +19,84 @@
     public static final String COMMAND_WORD = "find";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all tasks whose titles contain any of "
-            + "the specified keywords (case-sensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+            + "the specified keywords (case-sensitive)\n"
+            + "or whose date is between specified time duration "
+            + "or whose tag list contains any of the specified tags.\n "
+            + "Parameters: [KEYWORD]... [a/EARLIEST_TIME] [b/LATEST_TIME] [t/TAG]...\n"
+            + "Example: " + COMMAND_WORD + " CS3230 Midterm a/04-04-17 0000 t/exam";
+    public static final String MESSAGE_TIME_CONSTRAINTS = "The time a task happens before should be later "
```

Comment:
Should use the message time constraint defined in task creation to be consistent. Phrasing is also confusing